gandalf.js

```diff
@@ -1,89 +1,94 @@
-const randomElement = array[Math.floor(Math.random() * array.length)];
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.